woocommerce/woocommerce-android
# Move sort filter card in the products screen to the RecyclerView

Closes: #3260 


### Description

Sort-Filter card was sticky in the products screen. 
This PR moves Sort-Filter card in the RecyclerView, so it's not sticky anymore. If you touch Sort-Filter card, you couldn't scroll the list before. You can scroll by tapping Sort-Filter card now.

### Testing instructions

- Go to the Products tab.
- Scroll down the product list. You should have enough products to be able to scroll.
- Notice Sort-Filter cards are going up by scrolling.

### Images/gif

| before  | after |
| ------------- | ------------- |
| ![before products scrolled](https://user-images.githubusercontent.com/2471769/163282192-2e1e0664-ddb0-47c8-a405-f922d7c491ab.png) | ![after products scrolled](https://user-images.githubusercontent.com/2471769/163282229-a23bf3cd-af35-4a88-be54-b3eee1087bf7.png) |
